sympy/polys/polytools.py

```diff
@@ -3773,6 +3775,174 @@ def nth_power_roots_poly(f, n):
 
         return r.replace(t, x)
 
+    def root_bounds(f):
+        """
+        Compute lower and upper bounds on the magnitude of the nonzero roots of
+        this polynomial.
```

Comment:
I think it would be good to clarify in this docstring somewhere that we are thinking about potentially non-real roots.